src/pages/redeemed.js

```diff
@@ -120,11 +119,8 @@ class RedeemedPage extends React.Component {
   }
 
   async componentDidMount() {
-    const { getLoggedInUser, client, code } = this.props;
-
-    getLoggedInUser().then(LoggedInUser => {
-      this.setState({ LoggedInUser });
-    });
+    const { LoggedInUser, client, code } = this.props;
+    this.setState({ LoggedInUser });
```

Comment:
If component gets updated later (which will happen because `LoggedInUser` is fetched after the mount) then we won't get the updated `LoggedInUser`. We don't need to store the user in the state anymore, we can rely on `this.props` directly.